### PR TITLE
Update the Tutorials pages to source data from new "v3" Google Sheets

### PR DESCRIPTION
**Pull request by sureshc**
The HocTutorials and HocBeyondTutorials were ported to the v3 format in #20937 and #20855.  Update Pegasus Tutorials pages to use the new database tables created from the new Google Sheets.

### CODE REVIEW

**Comment by sureshc** [comment]
@breville Now that I have a manual process for carrying out regression tests on the Google Sheet import process, I'm considering reviving this logic that conditionally imports a "v3" Google Sheet without the datatype suffix on the column names:  https://github.com/code-dot-org/code-dot-org/blob/6410f12c632c1811a51feba2fea326feb1aee27b/pegasus/rake/seed.rake#L49-L119

Any thoughts on the tradeoffs with that approach?

**Comment by sureshc** [comment]
Looks like I need to resolve some issues because `test_tutorials.rb` is failing

**Comment by sureshc** [comment]
@breville Tests are passing now.  This is ready for you to review, using the technical design we discussed this morning.

File: pegasus/src/database.rb [hunk 1]
```diff
@@ -5,9 +5,23 @@
 require 'securerandom'
 
 class Tutorials
+  # The Tutorials pages used to source data from the tutorials and beyond_tutorials tables, which were imported from
+  # the GoogleDrive://Pegasus/Data/ HocTutorials and HocBeyondTutorials Google Sheets
```
Comment (breville) [inline pegasus/src/database.rb:9]: nit: periods at end of sentences :)

**Comment by sureshc** [comment]
I'll take a look at how recently the legacy Data/HocTutorials and Data/BeyondHocTutorials Google Sheets were modified to see if we need to refresh the content of the v3 copies of these sheets.

**Pull request merged**